packages/amplify-e2e-tests/overrides/override-storage-ddb.ts

```diff
@@ -0,0 +1,6 @@
+export function overrideProps(props: any) {
```

Comment:
nit: use a better type for `props`. Is there a CDK or Cloudform type we can use? If not, can at least use $TSObject

```suggestion
import { $TSObject } from 'amplify-cli-core';

export function overrideProps(props: $TSObject) {
```

Reply:
We'd be exporting these prop types to a new package - amplify-cli-helpers (which would be published in the future to npm)